pkg/manager/status/status_manager.go

```diff
@@ -173,6 +178,52 @@ func checkValidation(ctx context.Context, fc *flowslatest.FlowCollector) metav1.
 	}
 }
 
+func checkLoki(ctx context.Context, c client.Client, fc *flowslatest.FlowCollector) metav1.Condition {
+	if !helper.UseLoki(&fc.Spec) {
+		return metav1.Condition{
+			Type:    LokiIssue,
+			Reason:  "Unused",
+			Status:  metav1.ConditionUnknown,
+			Message: "Loki is disabled",
+		}
+	}
+	if fc.Spec.Loki.Mode != flowslatest.LokiModeLokiStack {
+		return metav1.Condition{
+			Type:    LokiIssue,
+			Reason:  "Unused",
+			Status:  metav1.ConditionUnknown,
+			Message: "Loki is not configured in LokiStack mode",
+		}
+	}
```

Comment:
How does that reports in the conditions ? :thinking: 

```
LokiIssue - Unused - Unknown - Loki is disabled
LokiIssue - Unused - Unknown - Loki is not configured in LokiStack
``` 

Maybe we could rename `LokiIssue` to `Loki` to report all the states as these are not issues.

Reply:
I don't think because what matters here is that it's a condition that needs to be either TRUE or FALSE (or UNKNOWN). Just "Loki" being true or false doesn't make sense, whereas having LokiIssue true or false does make sense.
When the condition is true, users see in the Console "LokiIssue" so they know there's something to check. If it was just "Loki", that would be unclear. When the condition is false, LokiIssue isn't displayed in the condensed status view, so that's fine

Reply:
See my previous PR https://github.com/netobserv/network-observability-operator/pull/1255 that was precisely about making conditions being "negative" so that they render better in the console

Reply:
Ok make sense !